add tutorial for sample granulation

HighHarmonics2 opened the pull request:
new tutorial added 
this is based on an example found in ~/Max 8/Projects/mercury/examples/07_granular-playback.txt

It seems like a good addition to the Tutorial series

tmhglnd commented:
Thanks for this, will give it a check soon! Just wanted to point out that the actual tutorials as stored as separate files here: https://github.com/tmhglnd/mercury-playground/tree/main/public/assets/tutorial 

The `.json` file is build from these textfiles when running `$ npm run data` or `$ npm run build`

So I'll probably move the example to an individual file, or you can do it when you have the time.

HighHarmonics2 commented:
Thanks. I see the tutorials now. I put this as a new chapter 8 only because I couldn't figure out where to put it. I think it is good to have more advanced examples. 

I reverted the first commit and added a new one.

tmhglnd commented:
I included the example, but also added a page to introduce chapter 8, so the granulation tutorial became 801.
